src/components/PushAudienceDialog/PushAudienceDialog.react.js

```diff
@@ -258,9 +263,9 @@ export default class PushAudienceDialog extends React.Component {
         </div>
         {futureUseSegment}
         <FormNote
-          show={Boolean(this.props.errorMessage && this.props.errorMessage.length > 0)}
+          show={Boolean(this.props.errorMessage && this.props.errorMessage.length > 0 || this.state.errorMessage && this.state.errorMessage.length > 0)}
```

Comment:
i'm not 100% sure it matters from operator precedence standpoint, but it would be easier to read if you put the two conditionals in parens:

```
(this.props.errorMessage && this.props.errorMessage.length > 0) || (this.state.errorMessage && this.state.errorMessage.length > 0)
```